pkg/ovndbcluster/volumes.go

```diff
@@ -58,6 +58,11 @@ func GetDBClusterVolumeMounts() []corev1.VolumeMount {
 			MountPath: "/var/lib/config-data",
 			ReadOnly:  false,
 		},
+		{
+			Name:      "etc-ovn",
+			MountPath: "/etc/ovn",
+			ReadOnly:  false,
+		},
```

Comment:
Does this also require a volume definition or is it able to somehow find the PVC created in the `VolumeClaimTemplate`?

Reply:
Yes volume definition is not required, just this is enough.